doc/content/reference/federated-auth/oidc/microsoft-entra.md

```diff
@@ -55,8 +55,8 @@ Register the provider. Set the following values
 ```bash
 OIDC_PROVIDER_ID="provider ID"                                # Provider ID from above.
 OIDC_PROVIDER_NAME="My OIDC Provider"                         # Name used to display on the Console.
-OIDC_CLIENT_ID="client123"                                    # Client ID is the Secret ID above.
-OIDC_CLIENT_SECRET="secret123"                                # Client Secret is the secret Value from above..
+OIDC_CLIENT_ID="client123"                                    # Client ID is the Application (client)ID from above.
```

Comment:
```suggestion
OIDC_CLIENT_ID="client123"                                    # Client ID is the Application (client) ID from above.
```

Reply:
We have updated the suggested change. Could you please merge it now?